Changing old API links (#6199)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-versions/2023-release-notes.md

    **Beginning June 1, 2023,** developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through the dbt Cloud user interface (UI) and API for only 365 days, on a rolling basis. Older run history will be available for download by reaching out to Customer Support. We're seeking to minimize the amount of metadata we store while maximizing application performance. 

>     Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#/) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  Additionally, the run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  

Reviewer: the link text and the link URL doesn't seem to match. is there still a `Runs` endpoint?

Author: Yes, but there is no landing page for it anymore. It will be on the sidebar. I guess we could just land on the first page in the runs list. I'll change it to that

>     Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#/) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  Additionally, the run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  

Author: ```suggestion
    Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2#/operations/List%20Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  Additionally, the run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  
```

    :::

>     For more info, refer to our [documentation](#tag/Runs/operation/listRunsForAccount).

Author: ```suggestion
    For more info, refer to our [documentation](https://docs.getdbt.com/dbt-cloud/api-v2#/operations/List%20Runs).
```

    This maximum limit applies to [multi-tenant instances](/docs/cloud/about-cloud/access-regions-ip-addresses) only, and _does not_ apply to single tenant instances.

>     Refer to the [Pagination](https://docs.getdbt.com/dbt-cloud/api-v2#/) section for more information on this change. 

Reviewer: i might be missing something. i don't see a `Pagination` section on https://docs.getdbt.com/dbt-cloud/api-v2#/

Author: You have to scroll down to see it on the page we link to. Can't anchor link to it like we could in the old APIs. I know Gideon is working on enhancements

Reviewer: oh i see. i was looking at the sidebar. 

as an idea, maybe "... **Pagination** section on the [API v2](https://docs.getdbt.com/dbt-cloud/api-v2#/) page ..." or something? 

totally non-blocking.

>     Refer to the [Pagination](https://docs.getdbt.com/dbt-cloud/api-v2#/) section for more information on this change. 

Author: ```suggestion
    Refer to the [Pagination](https://docs.getdbt.com/dbt-cloud/api-v2#/) section of the overview for more information on this change. 
```
